package.json

```diff
@@ -4,50 +4,63 @@
   "description": "Blazing Fast React UI Primitive",
   "contributors": [
     "Jeroen Ransijn (https://twitter.com/jeroen_ransijn)",
-    "Roland Warmerdam (https://roland.codes)"
+    "Roland Warmerdam (https://roland.codes)",
+    "Meichen Zhou (https://github.com/jfmaggie)",
+    "Netto Farah (https://twitter.com/nettofarah)"
   ],
   "keywords": [
     "react"
   ],
   "repository": "segmentio/ui-box",
   "license": "MIT",
-  "main": "lib/index.js",
+  "main": "lib/src/index.js",
```

Comment:
I think you need to add a `types` field too? https://www.typescriptlang.org/docs/handbook/declaration-files/publishing.html#including-declarations-in-your-npm-package